graphdatascience/model/kge_runner.py

```diff
@@ -0,0 +1,261 @@
+import json
+import logging
+import os
+import time
+from typing import Any, Dict, Optional
+
+import pandas as pd
+import requests
+from pandas import DataFrame, Series
+
+from ..error.client_only_endpoint import client_only_endpoint
+from ..error.illegal_attr_checker import IllegalAttrChecker
+from ..error.uncallable_namespace import UncallableNamespace
+from ..graph.graph_object import Graph
+from ..query_runner.query_runner import QueryRunner
+from ..server_version.server_version import ServerVersion
+
+logging.basicConfig(level=logging.INFO)
+
+
+class KgeRunner(UncallableNamespace, IllegalAttrChecker):
+    def __init__(
+        self,
+        query_runner: QueryRunner,
+        namespace: str,
+        server_version: ServerVersion,
+        compute_cluster_ip: str,
+        encrypted_db_password: str,
+        arrow_uri: str,
+    ):
+        self._query_runner = query_runner
+        self._namespace = namespace
+        self._server_version = server_version
+        self._compute_cluster_web_uri = f"http://{compute_cluster_ip}:5005"
+        self._compute_cluster_mlflow_uri = f"http://{compute_cluster_ip}:8080"
+        self._encrypted_db_password = encrypted_db_password
+        self._arrow_uri = arrow_uri
+
+    @property
+    def model(self):
+        return self
+
+    # @compatible_with("stream", min_inclusive=ServerVersion(2, 5, 0))
+    @client_only_endpoint("gds.kge.model")
+    def train(
```

Comment:
So these are somehow excluded from documentation?
If not, then I think the sphinx refdoc will complain about missing documentation for it because of `test_client_only_endpoints.py`. Maybe the workaround is to exclude these in that script

Reply:
ok, thanks for this observation